tardis/plasma/tests/test_plasma_dot_tex.py

```diff
@@ -0,0 +1,20 @@
+import os
+import pygraphviz as pgv
+import pytest
+
+data_path = os.path.join('tardis', 'plasma', 'tests', 'data')
+
+
+def test_write_dot(tmpdir, simulation_verysimple):
+    fname = str(tmpdir.mkdir('test_dot').join('plasma.dot'))
+    simulation_verysimple.plasma.write_to_dot(fname)
+    actual = pgv.AGraph(fname).to_string()
+    expected = pgv.AGraph(os.path.join(data_path, 'plasma_ref.dot')).to_string()
+    assert actual == expected
```

Comment:
@wkerzendorf This passes on my Travis [build](https://travis-ci.org/vg3095/tardis/builds/267434622) for Ubuntu , not for Mac . And on Tardis Travis CI it fails for Ubuntu as well as for Mac. I can't understand the reason for this.